rviz_common/include/rviz_common/config.hpp

```diff
@@ -133,9 +136,6 @@ class RVIZ_COMMON_PUBLIC Config
   /// The default constructor creates an empty config object.
   Config();
 
-  /// The copy constructor copies only the reference to the data, not the data itself.
-  Config(const Config & source);
```

Comment:
@richmattes Is the other way to fix this to also add an overload for the `=` operator?